README.md

```diff
@@ -13,14 +13,17 @@
 
 ## Table of Contents
 <!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
-- [Image content](#image-content)
+- [Notice: Dockerhub repository has been migrated to lycheeorg/lychee](#notice-dockerhub-repository-has-been-migrated-to-lycheeorglychee)
```

Comment:
At this point we can probably drop this notice.

Reply:
[48c2f3c](https://github.com/LycheeOrg/Lychee-Docker/pull/216/commits/48c2f3c066ce0f4b1249a222d1e7be6cc55473a7)